index.bs

```diff
@@ -427,12 +425,18 @@ To <dfn>parse an attribution destination</dfn> from a string |str|:
 
 <h3 algorithm id="noising-attribution">Noising attribution</h3>
 
-To <dfn>select a random attribution mode</dfn> given a double |noiseRate|, run the following steps:
+To <dfn>select a random attribution mode</dfn> given doubles |neverRate| and |falselyRate|, run the
+following steps:
 
-1. Assert: |noiseRate| is between 0 and 1 (both inclusive).
+1. Assert: |neverRate| is between 0 and 1 (both inclusive).
+1. Assert: |falselyRate| is between 0 and 1 (both inclusive).
+1. Let |noiseRate| be the sum of |neverRate| and |falselyRate|.
+1. Assert: |noiseRate| <= 1.
 1. Let |r| be a random double between 0 (inclusive) and 1 (exclusive).
 1. If |r| is greater than or equal to |noiseRate|, return "<code>truthfully</code>".
-1. Return "<code>never</code>".
+1. Let |t| be a random double between 0 (inclusive) and 1 (exclusive).
+1. If |t| is greater than or equal to |falselyRate|, return "<code>never</code>".
```

Comment:
Should t be a random number between 0 and noiseRate here?

Reply:
Would it be less confusing to just specify this algorithm as:

> Return one of "`truthfully`", "`falsely`", or "`never`", with implementation-defined probability.

?

Reply:
Yes I am supportive of that simplification (seeing this version of the algorithm).